docsets/PyTorch/docset.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "PyTorch",
-    "version": "1.0.0",
+    "version": "1.1.0",
     "archive": "PyTorch.tgz",
```

Comment:
Note: next time please update the main archive as well, not just `specific_versions`. I'll fix it this time.

Reply:
Thanks.